CMS-Enterprise/mint-app
# [MINT-3230] About this milestone panel

# MINT-3230

## Description


- Added `<MilestonePanel />`
- Added `<SolutionCard />`
- Tweaked styles and functionality of solution panels
- Added link to render/add custom milestone form
- Added unit tests


## How to test this change

1.  Create model plan and navigate to MTO -> milestone library
2. Click on any 'About this milestone'
3. Verify figma matches
4. Verify responsiveness
5. Verify solution panels work as expected/overlay on top of milestone
6. Verify the add custom milestone works

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
